:iphone:  Add alternate urls to Samsung

adriens opened the pull request:
Like for [Nokia](https://github.com/endoflife-date/endoflife.date/blob/master/products/nokia.md) : 

```yaml
alternate_urls:
-   /hmd
-   /nokiamobile
```

:moneybag: This will make it easier to retrieve samsung devices : 

<pre><font color="#5FD700">❯</font> eol nokia | head -10
| cycle                |  release   |    eol     | link                                                      |
|:---------------------|:----------:|:----------:|:----------------------------------------------------------|
| Nokia C21            | 2022-05-03 |   False    | https://www.nokia.com/phones/en_int/nokia-c-21            |
| Nokia C21 Plus       | 2022-04-29 |   False    | https://www.nokia.com/phones/en_int/nokia-c-21-plus       |
| Nokia C2 2nd Edition | 2022-04-19 | 2024-05-01 | https://www.nokia.com/phones/en_int/nokia-c-2-2nd-edition |
| Nokia G11            | 2022-03-24 | 2025-05-01 | https://www.nokia.com/phones/en_int/nokia-g-11            |
| Nokia G21            | 2022-02-15 | 2025-03-01 | https://www.nokia.com/phones/en_int/nokia-g-21            |
| Nokia T20            | 2021-11-02 | 2024-11-01 | https://www.nokia.com/phones/en_int/nokia-t-20            |
| Nokia G50            | 2021-10-13 | 2024-12-01 | https://www.nokia.com/phones/en_int/nokia-g-50            |
| Nokia C30            | 2021-10-12 | 2023-10-01 | https://www.nokia.com/phones/en_int/nokia-c-30            |
<font color="#5FD700">❯</font> eol samsung
Product not found, run &apos;eol all&apos; for list</pre>

adriens committed:
Add alternate urls

Like for Nokia

usta commented on products/samsungmobile.md:
```diff
@@ -1,5 +1,8 @@
 ---
 permalink: /samsung-mobile
+alternate_urls:
+-   /samsung
```
-1 for me 👎 
Samsung is a company , this looks like adding microsoft as url to windows

adriens committed:
Only keep samsungmobile (remove samsung)

As of https://github.com/endoflife-date/endoflife.date/pull/2401#issuecomment-1409188885
